src/test/java/com/zaxxer/hikari/pool/TestMetrics.java

```diff
@@ -16,289 +16,18 @@
 
 package com.zaxxer.hikari.pool;
 
-import static com.zaxxer.hikari.pool.TestElf.newHikariConfig;
-import static com.zaxxer.hikari.pool.TestElf.newHikariDataSource;
-import static com.zaxxer.hikari.util.UtilityElf.quietlySleep;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-import static org.junit.Assume.assumeFalse;
-
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.util.SortedMap;
-import java.util.concurrent.TimeUnit;
-
+import com.zaxxer.hikari.HikariDataSource;
 import org.junit.Test;
 
-import com.codahale.metrics.Histogram;
-import com.codahale.metrics.Metric;
-import com.codahale.metrics.MetricFilter;
-import com.codahale.metrics.MetricRegistry;
-import com.codahale.metrics.Timer;
-import com.codahale.metrics.health.HealthCheck.Result;
-import com.codahale.metrics.health.HealthCheckRegistry;
-import com.zaxxer.hikari.HikariConfig;
-import com.zaxxer.hikari.HikariDataSource;
-import com.zaxxer.hikari.metrics.MetricsTrackerFactory;
-import com.zaxxer.hikari.metrics.dropwizard.CodahaleMetricsTrackerFactory;
-import com.zaxxer.hikari.util.UtilityElf;
+import static com.zaxxer.hikari.pool.TestElf.newHikariDataSource;
 
 /**
- * Test HikariCP/CodaHale metrics integration.
+ * Test HikariCP metrics integration.
  *
  * @author Brett Wooldridge
  */
 public class TestMetrics
```

Comment:
Logically, this got split into to `TestMetricsBase` and `CodahaleMetricsTest`.  Retained here is the single Codahale/Dropwizard-independent test (for the fake registry).